ui/packages/consul-ui/app/components/topology-metrics/card/index.scss

```diff
@@ -74,5 +74,37 @@
   }
   .details {
     padding: 0 12px 12px 12px;
+    > *:not(:last-child) {
+      padding-bottom: 6px;
+    }
+    span {
+      display: none;
+    }
```

Comment:
Do you need this? Is it targeting the same span below where its set to `inline-block`?

Reply:
not needed. removed.